modules/selenium/src/test/java/org/testcontainers/junit/SpecificImageNameWebDriverContainerTest.java

```diff
@@ -0,0 +1,28 @@
+package org.testcontainers.junit;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.testcontainers.containers.BrowserWebDriverContainer;
+
+import java.io.IOException;
+
+/**
+ *
+ */
+public class SpecificImageNameWebDriverContainerTest extends BaseWebDriverContainerTest {
+
+    @Rule
+    public BrowserWebDriverContainer firefox = new BrowserWebDriverContainer("selenium/standalone-firefox-debug:2.53.1-beryllium")
+            .withDesiredCapabilities(DesiredCapabilities.firefox());
+
+    @Test
```

Comment:
The `@Test` methods? We've not done this yet in the other subclasses. I guess this would require making the container field be a protected member of the parent class and initialize it in the subclass constructors. Or is there a simpler way?